tests/v2/validation/Jenkinsfile

```diff
@@ -94,6 +95,14 @@ node {
           dir ("./") {
             try {
               stage('Configure and Build') {
+                config = config.replace('${AWS_SECRET_ACCESS_KEY}', env.AWS_SECRET_ACCESS_KEY)
+                config = config.replace('${AWS_ACCESS_KEY_ID}', env.AWS_ACCESS_KEY_ID)
+                config = config.replace('${AWS_IAM_PROFILE}', env.AWS_IAM_PROFILE)
+                config = config.replace('${AWS_REGION}', env.AWS_REGION)
+                config = config.replace('${AWS_VPC}', env.AWS_VPC)
+                config = config.replace('${AWS_SECURITY_GROUPS}', env.AWS_SECURITY_GROUPS)
+                config = config.replace('${ADMIN_PASSWORD}', env.ADMIN_PASSWORD)
+
```

Comment:
I'm only concern is that my default we are using the Jenkins QA creds, which is something we don't necessarily want. @bmdepesa based on our future efforts to decouple creds. Is this something we want?